content/docs/setup/kubernetes/download/index.md

```diff
@@ -51,3 +51,51 @@ To use the Istio release Helm chart repository, add the Istio release repository
 {{< text bash >}}
 $ helm repo add istio.io https://storage.googleapis.com/istio-release/releases/{{< istio_full_version >}}/charts/
 {{< /text >}}
+
+## `istioctl` auto-completion
+
+If you are using Bash or Zsh, `istioctl` includes a tab completion file that you can use. 
+
+### Option 1. Installing `istioctl` on Linux
+
+Both the Bash and Zsh completion files are located in the `tools` directory. Copy the preferred completion file to your home directory, and add the following line to source the `istioctl` completion file in your `.bashrc` file. For example with Bash, you will be adding the following line:
+
+{{< text bash >}}
+$ source ~/istioctl.bash
+{{< /text >}}
+
+### Option 2. Installing `istioctl` on OSX
+
+If you are using OSX with the Bash shell, make sure that the bash-completion package is installed. If you are using [Homebrew](https://brew.sh), you can check to see if the bash-completion package is installed with the following command:
+
+{{< text bash >}}
+$ brew info bash-completion
+{{< /text >}}
+
+If installed, the output from this command should display output similar to the following:
+
+{{< text bash >}}
+$ brew info bash-completion
+bash-completion: stable 1.3 (bottled)
+{{< /text >}}
+
+If it is _not_ installed, proceed with installing the bash-completion package with the following command:
+
+{{< text bash >}}
+$ brew install bash-completion
+{{< /text >}}
+
+{{< tip >}}
+Once the auto-completion package has been installed on your OSX system, also make sure you add the following line of code to your `~/.bash_profile` file:
+
+`[ -f /usr/local/etc/bash_completion ] && . /usr/local/etc/bash_completion`
```

Comment:
`brew info bash-completion` suggests a different command for me:
```
==> Caveats
Add the following line to your ~/.bash_profile:
  [[ -r "/usr/local/etc/profile.d/bash_completion.sh" ]] && . "/usr/local/etc/profile.d/bash_completion.sh"

Bash completion has been installed to:
  /usr/local/etc/bash_completion.d
```

Reply:
@frankbu thanks for testing. I'll update this.